templates/zerver/developer-community.md

```diff
@@ -0,0 +1,71 @@
+- [#](https://chat.zulip.org/#narrow/stream/3-backend)[backend](https://chat.zulip.org/#narrow/stream/3-backend), [#](https://chat.zulip.org/#narrow/stream/6-frontend)[frontend](https://chat.zulip.org/#narrow/stream/6-frontend), and app-specific streams ([#](https://chat.zulip.org/#narrow/stream/48-mobile)[mobile](https://chat.zulip.org/#narrow/stream/48-mobile), etc.) are for getting help with various categories of work.
+## Streams for translators
+- [#translation](https://chat.zulip.org/#narrow/stream/58-translation) is for discussing Zulip’s translations.
+- [#translation/fr](https://chat.zulip.org/#narrow/stream/371-translation.2Ffr) and [#translation/zh_tw](https://chat.zulip.org/#narrow/stream/377-translation.2Fzh_tw) are for French and Traditional Chinese (Taiwan) translations, respectively.
+## Streams for anyone spending time in the community
+- [#announce](https://chat.zulip.org/#narrow/stream/1-announce) is our low-traffic stream for project announcements (releases, etc.).
+- [#checkins](https://chat.zulip.org/#narrow/stream/65-checkins) is for progress updates on what you’re working on and its status; usually folks post with their name as the topic. Everyone is welcome to participate!
+- [#design](https://chat.zulip.org/#narrow/stream/101-design) is where we discuss UI and feature design and collect feedback on potential design changes. We love feedback, so don’t hesitate to speak up! Note that [#feedback](https://chat.zulip.org/#narrow/stream/137-feedback) is usually the best place for end users to start a topic about a proposed design change; [#design](https://chat.zulip.org/#narrow/stream/101-design) is primarily intended for discussing changes the community is actively working on.
+- [#](https://chat.zulip.org/#narrow/stream/277-discussions)[discussions](https://chat.zulip.org/#narrow/stream/277-discussions) is where we do fun community conversations like “Ask Me Anything” Q&A sessions with project members.
+- [#documentation](https://chat.zulip.org/#narrow/stream/19-documentation) is where we discuss improving Zulip’s user, sysadmin, API, and developer documentation.
+- [#](https://chat.zulip.org/#narrow/stream/2-general)[general](https://chat.zulip.org/#narrow/stream/2-general) is for topics of general interest
+- [#learning](https://chat.zulip.org/#narrow/stream/92-learning) is for posting great learning resources one comes across.
+- [#](https://chat.zulip.org/#narrow/stream/97-off-topic)[off topic](https://chat.zulip.org/#narrow/stream/97-off-topic) is is for occasional conversations not related to Zulip, usually things of general interest to open-source communities.
```

Comment:
We need to fix the fact that Paper exports generate multiple copies of some links, like this one.